core/Db/Schema/Mysql.php

```diff
@@ -334,6 +334,24 @@ public function getTablesCreateSql()
                                       ) $tableOptions
             ",
 
+            'archiving_metrics'     => "CREATE TABLE {$prefixTables}archiving_metrics (
+                                      metadataid BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                      idarchive BIGINT UNSIGNED NULL,
+                                      idsite INTEGER UNSIGNED NOT NULL,
+                                      segment VARCHAR(255) NULL,
```

Comment:
```suggestion
                                      segment VARCHAR(255) NOT NULL,
```

At least in the current implementation this will never be `null`.

Reply:
I implemented your suggestion of adding the done* flag